tests/queries/models.py

```diff
@@ -449,6 +449,10 @@ def __str__(self):
         return "category item: " + str(self.category)
 
 
+class MixedCaseCategoryItem(models.Model):
+    Category = models.ForeignKey('SimpleCategory', models.CASCADE)
```

Comment:
Add a comment to draw attention to the capital C, or maybe even change the name to something like `CaTeGoRy`. It's quite easy to miss it as it stands.

Reply:
either that or leave the attribute name alone and define a `db_column='CaTeGoRy'` to put more emphasis on it.

Reply:
Actually I think the test with `db_column='CaTeGoRy'` is a slightly different test, and one we may want to include as well.

Reply:
Yes, it's a different case. I added it.